app/filters.php

```diff
@@ -68,3 +73,22 @@
     );
     return template_path(locate_template(["views/{$comments_template}", $comments_template]) ?: $comments_template);
 }, 100);
+
+/**
+ * Render WordPress searchform using Blade
+ */
+add_filter('get_search_form', function () {
+    return template('partials.searchform');
+});
+
+/**
+ * Collect data for searchform.
+ */
+add_filter('sage/template/app/data', function ($data) {
+    $data['sf_action'] = esc_url(home_url('/'));
+    $data['sf_screen_reader_text'] = _x('Search for:', 'label', 'sage');
+    $data['sf_placeholder'] = esc_attr_x('Search &hellip;', 'placeholder', 'sage');
+    $data['sf_current_query'] = get_search_query();
+    $data['sf_submit_text'] = esc_attr_x('Search', 'submit button', 'sage');
+    return $data;
```

Comment:
```php
return $data + [
    'sf_action' => esc_url(home_url('/')),
    // etc...
];
```

That will preserve `sf_*` key-value pairs that are already present in the `$data` array and only merge them in if they don't already exist.

Reply:
Should be all set